src/EFCore.Design/Design/Internal/CSharpHelper.cs

```diff
@@ -737,6 +737,71 @@ public virtual string Literal(object?[,] values)
         return builder.ToString();
     }
 
+    /// <summary>
+    ///     This is an internal API that supports the Entity Framework Core infrastructure and not subject to
+    ///     the same compatibility standards as public APIs. It may be changed or removed without notice in
+    ///     any release. You should only use it directly in your code with extreme caution and knowing that
+    ///     doing so can result in application failures when updating to a new Entity Framework Core release.
+    /// </summary>
+    public virtual string Literal<T>(IList<T> values, bool vertical = false)
```

Comment:
```suggestion
    public virtual string Literal<T>(List<T> values, bool vertical = false)
```
I think these overloads should be very specific to the type of literal they generate. There are lots of types that implement IList\<T>

Reply:
Fine with me.